lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -229,32 +224,54 @@ bool ProxyRenderDelegate::requiresUpdate(const MSubSceneContainer& container, co
     return true;
 }
 
+void ProxyRenderDelegate::_ClearRenderDelegate()
+{
+    // The order of deletion matters. Some orders cause crashes.
+    // This order matches the deletion order when a ProxyRenderDelegate is destroyed.
+    // Class member variables are destroyed in the reverse order in which they appear in the class declaration.
+
+    _sceneDelegate.reset(nullptr);
```

Comment:
I would expect that you don't have to be explicit about `nullptr` here and can just call `reset()`